apps/web/src/pages/Profile/ExperienceFormPage/components/ExperienceSkills.tsx

```diff
@@ -80,139 +65,158 @@ const ExperienceSkills = ({ skills, pool }: ExperienceSkillsProps) => {
-              ))}
-            </div>
-          )}
-          {technicalNonessentialSkills && (
-            <div
-              data-h2-radius="base(rounded)"
-              data-h2-shadow="base(s)"
-              data-h2-padding="base(x.5, x1, x.5, x.5)"
-            >
-              <span data-h2-font-weight="base(700)">
+              </li>
+              <li>
+                {intl.formatMessage({
+                  defaultMessage:
+                    "What tasks or activities did you do that relate to this skill?",
+                  id: "ibT7BK",
+                  description:
+                    "Question list item in link featured skills section of experience form (tasks).",
+                })}
+              </li>
+              <li>
+                {intl.formatMessage({
+                  defaultMessage:
+                    "Were there any special techniques or approaches that you used?",
+                  id: "x2xN5X",
+                  description:
+                    "Question list item in link featured skills section of experience form (techniques).",
+                })}
+              </li>
+              <li>
+                {intl.formatMessage({
+                  defaultMessage:
+                    "How much responsibility did you have in this role?",
+                  id: "u/IV5p",
+                  description:
+                    "Question list item in link featured skills section of experience form (responsibilities).",
+                })}
+              </li>
+            </ul>
+          </Accordion.Content>
+        </Accordion.Item>
+      </Accordion.Root>
+      {derivedType ? (
+        <div
+          data-h2-display="base(flex)"
+          data-h2-gap="base(x1)"
+          data-h2-flex-direction="base(column)"
+          data-h2-flex-wrap="p-tablet(wrap)"
+          data-h2-margin-top="base(x1)"
+        >
+          <div data-h2-align-self="base(flex-end)">
+            <SkillDialog
+              trigger={{
+                label: intl.formatMessage({
+                  defaultMessage: "Add a skill",
+                  id: "Oeb04k",
+                  description:
+                    "Label for skill dialog trigger on experience skills section.",
+                }),
+              }}
+              context="experience"
+              skills={skills}
+              onSave={handleAddSkill}
+            />
+          </div>
+          {watchedSkills && watchedSkills.length > 0 ? (
+            <SkillsInDetail
+              skills={fields as FormSkills}
+              onDelete={handleRemoveSkill}
+            />
+          ) : (
+            <Well>
+              <p
+                data-h2-margin-bottom="base(x1)"
+                data-h2-text-align="base(center)"
+                data-h2-font-weight="base(bold)"
+              >
+                {intl.formatMessage({
+                  defaultMessage:
+                    "You haven’t featured any skills on this experience yet.",
```

Comment:
Found a smart apostrophe😅 

```suggestion
                    "You haven't featured any skills on this experience yet.",
```